docs/reference/esql/functions/layout/qstr.asciidoc

```diff
@@ -0,0 +1,17 @@
+// This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+
+[discrete]
+[[esql-qstr]]
+=== `QSTR`
+
+preview::["Do not use `VALUES` on production environments. This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
```

Comment:
It is weird `VALUES` is mentioned here, should be `QSTR`.

Reply:
This was due to the docs being generated without the merged version from main - fixed in 28b72f9e945ca27a824c6bb602a5e6e0e9f47bdf